config/var_qa.env

```diff
@@ -0,0 +1,12 @@
+DOCS_BASE_PATH=/api
+DOCS_EXTERNAL_DESCRIPTION="Documentation for using the Platform APIs"
+DOCS_EXTERNAL_URL=https://docs.google.com/document/d/1p3q9OT9latXqON20WDh4CNPxIShUunfGgqT163r-Caw/edit?usp=sharing
+DOCS_HOST=qa-platform.nypl.org
+DOCS_INFO_DESCRIPTION="Endpoints require an OpenID Connect token from the NYPL OAuth server and interact with NYPL Data Streams."
+DOCS_INFO_TITLE="NYPL Platform APIs"
+DOCS_INFO_VERSION=qa
+DOCS_S3_BUCKET=qa-platformdocs.nypl.org
+DOCS_S3_CACHE_SECONDS=0
+DOCS_S3_KEY=api/list.json
+DOCS_URLS=https://qa-platform.nypl.org/docs/patron-creator,https://qa-platform.nypl.org/docs/patron-validator,https://qa-platform.nypl.org/docs/book-lists,https://qa-platform.nypl.org/docs/doc,https://qa-platform.nypl.org/docs/patron,https://qa-platform.nypl.org/docs/job,https://qa-platform.nypl.org/docs/recap-bibs,https://qa-platform.nypl.org/docs/bib,https://qa-platform.nypl.org/docs/item,https://qa-platform.nypl.org/docs/discovery,https://qa-platform.nypl.org/docs/hold-requests
+SLACK_TOKEN=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAIgwgYUGCSqGSIb3DQEHBqB4MHYCAQAwcQYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAzRAeBrmM2k++YPQaQCARCARFMUqaBR36TwOo19BfxOlj9JRC9KELK7bRscjRRthFnd0VuSY+cCo87I/1KlDo/DfvNX9MwwxMVNSZqS15XL5L1BeaT7
```

Comment:
I wonder if we should add a `SLACK_USERNAME=docsService-qa` to differentiate it from the `SLACK_USERNAME=docsService` given in `config/var_app`

Reply:
Done